causalml/inference/tree/causaltree.pyx

```diff
@@ -110,7 +111,7 @@ cdef class CausalMSE(RegressionCriterion):
         ct_var = ((node_sq_sum - node_tr_sq_sum) / node_ct -
                   (node_sum - node_tr_sum) * (node_sum - node_tr_sum) / (node_ct * node_ct))
 
-        return node_tau * node_tau - (tr_var / node_tr + ct_var / node_ct)
+        return big_number - (node_tau * node_tau - (tr_var / node_tr + ct_var / node_ct))
```

Comment:
We don't need a constant term because it will be canceled out when calculating the improvement in impurity. Let's just flip the sign.